cmake/Modules/Findmurmurhash2.cmake

```diff
@@ -0,0 +1,44 @@
+# - Try to find murmurhash2 headers and libraries.
```

Comment:
MurmurHash2 is not delivered as a separate library and this file has no use.

Reply:
its needed for this macro - find_contrib_lib(murmurhash2) in UNBUNDLED mode.
other solution - do not use find_contrib_lib